### ISSUE TITLE
Error in plugin generation flow with renaming README example

### ISSUE THREAD

**Issue by gazay** [postcss/postcss-plugin-boilerplate#5]
I followed steps from Readme but I received error. Here is a log: https://gist.github.com/gazay/bddc651835cb1950ea75

**Comment by gazay** [comment]
It seems that this error is not constant repetitive, maybe `Sync` methods not so sync

**Comment by MattDiMu** [comment]
I got the same issue (every time, however).

**Comment by ai** [comment]
@MattDiMu what is your OS, node version?

Can you take me your stacktrace (maybe error is difference, because @gazay has it not every time).

**Comment by MattDiMu** [comment]
You're right, I judged too quickly. I'm using OS X Yosemite 10.10.3 and the error message is slightly different. No README.md file was created/renamed. I hope nothing else is missing :)

````
Error: ENOENT, no such file or directory '/Users/matthias/GitHub/postcss-plugin-boilerplate/README.md.example'
    at Error (native)
    at Object.fs.renameSync (fs.js:638:18)
    at cleanRepo (/Users/matthias/GitHub/postcss-plugin-boilerplate/start:116:8)
    at nextQuestion (/Users/matthias/GitHub/postcss-plugin-boilerplate/start:83:9)
    at questionCallback (/Users/matthias/GitHub/postcss-plugin-boilerplate/start:59:21)
    at ReadStream.<anonymous> (/Users/matthias/GitHub/postcss-plugin-boilerplate/start:13:5)
    at ReadStream.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at ReadStream.Readable.push (_stream_readable.js:126:10)
    at TTY.onread (net.js:529:20)


````

**Comment by ai** [comment]
Hm. Can you help me to investigate source of problem?

Does repo contains README.md.example after clone? Maybe in wrong letter case?

**Comment by ai** [comment]
Next test: can you comment 82 line in `start` file with `fillFiles(__dirname);` and run `start`?

**Comment by MattDiMu** [comment]
README.md.example wasn't in the repo at all. 

After commenting line 82 and running the wizard, I didn't get any error message. The directory was renamed (which is wasn't before) and a README was present. It contains some placeholders like "PostCSS] plugin PLUGIN_DESC.", however.

**Comment by ai** [comment]
How `README.md.example` can be missed in repo? O_o do you check it on fresh `git clone`?

**Comment by MattDiMu** [comment]
yes, i'm sure. I only copy-pasted the 2 commands from the first 2 steps (and i was/am in an freshly created directory)

**Comment by ai** [comment]
@MattDiMu but does `README.md.example` exists after 1 comand? Not after `start`?

**Comment by ai** [comment]
@MattDiMu ping :)

**Comment by MattDiMu** [comment]
@ai yes both readme exist (after cloning the repo)

**Comment by ai** [comment]
OK. Next test.

1. Clone fresh dir.
2. Comment 102-108 lines in `start`.
3. Run `start` again.

We will find this mistery bug!

**Comment by MattDiMu** [comment]
Sorry for the delay - I'm not working every we :)

````
fs.js:638
  return binding.rename(pathModule._makeLong(oldPath),
                 ^
Error: ENOENT, no such file or directory '/Users/matthias/GitHub/test2/postcss-plugin-boilerplate/README.md.example'
    at Error (native)
    at Object.fs.renameSync (fs.js:638:18)
    at cleanRepo (/Users/matthias/GitHub/test2/postcss-plugin-boilerplate/start:118:8)
    at nextQuestion (/Users/matthias/GitHub/test2/postcss-plugin-boilerplate/start:83:9)
    at questionCallback (/Users/matthias/GitHub/test2/postcss-plugin-boilerplate/start:59:21)
    at ReadStream.<anonymous> (/Users/matthias/GitHub/test2/postcss-plugin-boilerplate/start:13:5)
    at ReadStream.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at ReadStream.Readable.push (_stream_readable.js:126:10)
    at TTY.onread (net.js:529:20)
````

**Comment by ai** [comment]
So error is same even if you comment `fs.writeFileSync(filepath, content);`?

Next turn:

1. Comment 94-110 lines
2. Comment 95-109
3. Comment 99 line
4. Comment 102-108.

All this steps should be down on clear copy. And could you please run `start` 3 be sure, that error is not missed by different process list on your CPU?

**Comment by MattDiMu** [comment]
I don't understand what u mean in your last sentence (different process list on my cpu), but i just cloned 4 fresh repos in different folders, commented/deleted the line numbers and got this:

1. no error
2. 
````
fs.js:638
  return binding.rename(pathModule._makeLong(oldPath),
                 ^
Error: ENOENT, no such file or directory '/Users/matthias/GitHub/test-2/postcss-plugin-boilerplate/README.md.example'
    at Error (native)
    at Object.fs.renameSync (fs.js:638:18)
    at cleanRepo (/Users/matthias/GitHub/test-2/postcss-plugin-boilerplate/start:116:8)
    at nextQuestion (/Users/matthias/GitHub/test-2/postcss-plugin-boilerplate/start:83:9)
    at questionCallback (/Users/matthias/GitHub/test-2/postcss-plugin-boilerplate/start:59:21)
    at ReadStream.<anonymous> (/Users/matthias/GitHub/test-2/postcss-plugin-boilerplate/start:13:5)
    at ReadStream.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at ReadStream.Readable.push (_stream_readable.js:126:10)
    at TTY.onread (net.js:529:20)
````

3.

````
fs.js:638
  return binding.rename(pathModule._makeLong(oldPath),
                 ^
Error: ENOENT, no such file or directory '/Users/matthias/GitHub/test-3/postcss-plugin-boilerplate/README.md.example'
    at Error (native)
    at Object.fs.renameSync (fs.js:638:18)
    at cleanRepo (/Users/matthias/GitHub/test-3/postcss-plugin-boilerplate/start:116:8)
    at nextQuestion (/Users/matthias/GitHub/test-3/postcss-plugin-boilerplate/start:83:9)
    at questionCallback (/Users/matthias/GitHub/test-3/postcss-plugin-boilerplate/start:59:21)
    at ReadStream.<anonymous> (/Users/matthias/GitHub/test-3/postcss-plugin-boilerplate/start:13:5)
    at ReadStream.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at ReadStream.Readable.push (_stream_readable.js:126:10)
    at TTY.onread (net.js:529:20)
````


4.

````
fs.js:638
  return binding.rename(pathModule._makeLong(oldPath),
                 ^
Error: ENOENT, no such file or directory '/Users/matthias/GitHub/test-4/postcss-plugin-boilerplate/README.md.example'
    at Error (native)
    at Object.fs.renameSync (fs.js:638:18)
    at cleanRepo (/Users/matthias/GitHub/test-4/postcss-plugin-boilerplate/start:116:8)
    at nextQuestion (/Users/matthias/GitHub/test-4/postcss-plugin-boilerplate/start:83:9)
    at questionCallback (/Users/matthias/GitHub/test-4/postcss-plugin-boilerplate/start:59:21)
    at ReadStream.<anonymous> (/Users/matthias/GitHub/test-4/postcss-plugin-boilerplate/start:13:5)
    at ReadStream.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at ReadStream.Readable.push (_stream_readable.js:126:10)
    at TTY.onread (net.js:529:20)
````

Looks like line 94 causes the problem for whatever reason?

**Comment by ai** [comment]
How many times do yo run every test? I think we have a randomm error here. And if we will run test only once, we will have false nigative result, when we doesn’t get a error, but it it here.

Can you run test 1 by 3 times?

**Comment by kristoferjoseph** [comment]
I am seeing this as well.

**Comment by ai** [comment]
@kristoferjoseph maybe you can look into this bug. I have no ideas right now :(.

**Comment by kristoferjoseph** [comment]
@ai I'll dig in. Feels like an async issue, but could be wrong. Seems like the README file is being written to before it is available.

**Comment by verybigman** [comment]
:+1: i get this error to :(

**Comment by ai** [comment]
@verybigman maybe you can try to debug it?

**Comment by verybigman** [comment]
@ai maybe later

**Comment by Galvo87** [comment]
Having the same issue here on Osx 10.10.4 :(

**Comment by ai** [comment]
@Galvo87 can you debug it?

**Comment by Galvo87** [comment]
@ai sure I can help, btw I'm a newbie developer :)

This is the error I get every time, even after cloning the fresh repo... it complains about README.md.example... I see it disappearing from the folder, maybe it gets deleted before it's used?

<pre>
<code>
fs.js:636
  return binding.rename(pathModule._makeLong(oldPath),
                 ^
Error: ENOENT, no such file or directory '/Volumes/hd/projects/postcss-mq-visualizer/postcss-plugin-boilerplate-master/README.md.example'
    at Error (native)
    at Object.fs.renameSync (fs.js:636:18)
    at cleanRepo (/Volumes/hd/projects/postcss-mq-visualizer/postcss-plugin-boilerplate-master/start:118:8)
    at nextQuestion (/Volumes/hd/projects/postcss-mq-visualizer/postcss-plugin-boilerplate-master/start:83:9)
    at questionCallback (/Volumes/hd/projects/postcss-mq-visualizer/postcss-plugin-boilerplate-master/start:59:21)
    at ReadStream.<anonymous> (/Volumes/hd/projects/postcss-mq-visualizer/postcss-plugin-boilerplate-master/start:13:5)
    at ReadStream.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at ReadStream.Readable.push (_stream_readable.js:126:10)
    at TTY.onread (net.js:538:20)
</code>
</pre>

**Comment by ai** [comment]
Seems like I fix it in eac614d

Can somebody test it again?
